internal/exec/terraform_outputs.go

```diff
@@ -321,21 +318,15 @@ func GetTerraformOutput(
 		// Execute `terraform output`
 		terraformOutputs, err := execTerraformOutput(atmosConfig, component, stack, sections)
 		if err != nil {
-			if atmosConfig.Logs.Level == u.LogLevelTrace || atmosConfig.Logs.Level == u.LogLevelDebug {
-				fmt.Printf("\r✗ %s\n", message)
-			}
-			l.Fatal("Failed to execute terraform output", "component", component, "stack", stack, "error", err)
+			u.PrintMessage(fmt.Sprintf("\r✗ %s\n", message))
+			l.Fatal("Failed to execute terraform output", "component", component, "stack", stack, errorKey, err)
 		}
 
 		// Cache the result
 		terraformOutputsCache.Store(stackSlug, terraformOutputs)
 		result = getTerraformOutputVariable(atmosConfig, component, stack, terraformOutputs, output)
 	}
-
-	if atmosConfig.Logs.Level == u.LogLevelTrace || atmosConfig.Logs.Level == u.LogLevelDebug {
-		// Show success
-		fmt.Printf("\r✓ %s\n", message)
-	}
+	u.PrintMessage(fmt.Sprintf("\r✓ %s\n", message))
```

Comment:
Does this go to stderr?

Reply:
Can we modify print message to support format specifiers so we don’t need to wrap sprintf?

Reply:
>Does this go to stderr?
https://github.com/cloudposse/atmos/blob/main/tests/test-cases/atmos-functions.yaml#L21-L26
Based on above test case we do not add the line to stderr.

> Can we modify print message to support format specifiers so we don’t need to wrap sprintf?
yep we can do that. On it

Reply:
This will break atmos ability to use terraform outputs because the json will get corrupted by this printing to stdout, causing a regression that we just (incorrectly) fixed. It should be fixed by outputting the console output to stderr, so stdout is the unmodified json.

Reply:
Done